packages/cypress/src/schematics/cypress-project/cypress-project.ts

```diff
@@ -68,11 +68,24 @@ function updateWorkspaceJson(options: CypressProjectSchema): Rule {
       }
     };
 
-    architect.lint = generateProjectLint(
-      normalize(options.projectRoot),
-      join(normalize(options.projectRoot), 'tsconfig.e2e.json'),
-      options.linter
-    );
+    architect.lint = options.linter
```

Comment:
You changed `['tslint', 'eslint']` to `['tslint', 'eslint', undefined]`, and you assigned `undefined` to mean `use build-angular`. I find it a bit too implicit.

I see the following options:

1. always use `build-angular` when selecting tslint
2. had a third option in the enum
3. always use nrwl/linter to lint cypress projects

I personally prefer 1 or 3. 

What do you think?

Reply:
I vote for 1

Reply:
Sounds good

Reply:
Sounds good